releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -14,6 +14,130 @@
 
 [Source code][source_code_web] | [Package (Maven)][package_web]
 
+## 5.9.0-beta.1 - January 11, 2024
+
+### Features Added
+
+* Added support for loading Feature Variants from Azure App Configuration.
+
+## 5.8.0 - December 13, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.0.13, 3.1.0-3.1.6, 3.2.0-3.2.0.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4, 2023.0.0-2023.0.0.
+
+### Features Added
+
+* Added support for Azure App Configuration Snapshots. Snapshots can be loaded by setting `spring.cloud.azure.appconfiguration.stores[0].snapshot-name` to the snapshot name. Snapshots can be created using the Azure CLI or the Azure Portal. See [here](https://docs.microsoft.com/azure/azure-app-configuration/concept-snapshots) for more information on snapshots.
+* Added support for trimming prefixes from keys,the default value is the key-filter when key-filter is used. The configuration is `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefix` and `spring.cloud.azure.appconfiguration.stores[0].trim-key-prefixes`. [#
+7598](https://github.com/Azure/azure-sdk-for-java/pull/37598)
```

Comment:
This link points to Snapshots PR.

Reply:
The snapshots PR actually was the thing that added this. `/application/` is a prefix on all of the keys in Spring. This was the first time this was added as the prefix needed to be removed in snapshots.